### DISCUSSION TITLE
Data sharing

### DISCUSSION THREAD

**Discussion by MichielHartsuiker** [telekom-security/tpotce#1251] [Q&A] [answered]
Hello, 

I'm a student and I am doing a research about honeypots for a company. So at the moment i'm testing tpot on a vm. But now I saw that the data is shared with the community and there is a way to stop sharing the data. Do I only need to remove this or is there something else that I need to remove or adjust? Because the company doesn't want to share there client data.

![image](https://user-images.githubusercontent.com/47798605/214515535-6e1a77e6-ed43-49c1-9553-e8fe0a0fb831.png)

Thanks in advance!

**Comment by t3chn0m4g3** [comment, marked as answer]
You are correct.

**Reply by MichielHartsuiker** [reply]
Thank you.
